mix.exs

```diff
@@ -61,7 +61,9 @@ defmodule Membrane.HTTPAdaptiveStream.MixProject do
       {:membrane_core, "~> 0.7.0"},
       {:membrane_cmaf_format, "~> 0.3.0"},
       {:ex_doc, "~> 0.21", only: [:dev, :test], runtime: false},
-      {:dialyxir, "~> 1.0.0-rc.7", only: [:dev, :test], runtime: false}
+      {:dialyxir, "~> 1.0.0-rc.7", only: [:dev, :test], runtime: false},
+      #! czy dla membrane_mp4_plugin dodać jakieś only: [:dev, :test] ? jest potrzebne tylko w moim binie
```

Comment:
Your bin is a part of a whole package so `membrane_mp4_plugin` is required for all environments. All those `only: ...` are necessary if some packages at some point are not required e.g. dialyxir checks if all your types in the package are well-fitted with each other, once you ship the package in production you probably don't need that therefore `only: [:dev, :test]`.